### PR TITLE
CS02 - testing

### PR DESCRIPTION
**Pull request by somya008**
testing

### CODE REVIEW

File: src/ServerRequests.java [hunk 1]
```diff
@@ -25,6 +25,7 @@
 		static Map<String,Integer> serverMap= new LinkedHashMap<String,Integer>();
 		//static Map<String,Integer> intialserverMap= new HashMap<String,Integer>();
 		String server_nm ="192.11.12";
+		// hello added
```
Comment (somya008) [inline src/ServerRequests.java]: remove this extra comment

File: src/ServerRequests.java [hunk 2]
```diff
@@ -25,6 +25,7 @@
 		static Map<String,Integer> serverMap= new LinkedHashMap<String,Integer>();
 		//static Map<String,Integer> intialserverMap= new HashMap<String,Integer>();
 		String server_nm ="192.11.12";
+		// hello added
 		int total_load = 0;
 		Integer i=3;
```
Comment (sharmidevgupta) [inline src/ServerRequests.java]: test

**Review by sharmidevgupta — CHANGES_REQUESTED** [review]

**Comment by vidhi08** [comment]
Closed - Opened for testing only

**Comment by vidhi08** [comment]
Closed - Opened for testing only

**Pull request closed without merging**